paper.bib

```diff
@@ -0,0 +1,77 @@
+@misc{frontier,
+    author = {Barb Helland},
+    title = {At the Frontier: DOE Supercomputing Launches the Exascale Era},
+    year = {2022},
+    publisher = {United States Department of Energy Office of Science},
+    journal = {Website},
+    url = {https://www.energy.gov/science/articles/frontier-doe-supercomputing-launches-exascale-era}
+}
```

Comment:
I think this can be considered an `article`, but anyways, "Website" is not a journal.
```suggestion
@article{frontier,
    author = {Barb Helland},
    title = {At the Frontier: DOE Supercomputing Launches the Exascale Era},
    year = {2022},
    publisher = {United States Department of Energy Office of Science},
    url = {https://www.energy.gov/science/articles/frontier-doe-supercomputing-launches-exascale-era}
}
```